components/coloring-listgroup.vue

```diff
@@ -55,23 +61,30 @@
 				// 1. Get the list group item element
```

Comment:
This is possible with the changeover to a `b-table` since usage of `tbody-tr-class` and `@row-selected` in conjunction is possible. The only stipulation here is that `currentCategory` should never be `null`. This is a UX consideration.

When a user is confronted with the categorization page, one category (the first, by default) should be selected as a visual prompt for the action of selecting categories from the listbox (or table).

On top of that, the state of not being able to paint doesn't currently make sense in our current design of the page.

Reply:
> The only stipulation here is that currentCategory should never be null

Now that I think about it, we currently treat `null` as an (empty) category. So there could be a use case for a no-current-category-selected state where `currentCategory` is `null` (e.g. you navigate back to categorization page to unassign a category but now first either need to select the current category to "unpaint" it or paint and "unpaint" with a different category).  But I agree, that might make the interface less intuitve. Besides, we have other ways of quickly unassigning a category (e.g. in the annotation page).